activestorage/app/models/active_storage/variation.rb

```diff
@@ -117,11 +118,7 @@ def processor
     end
 
     def pass_transform_argument(command, method, argument)
-      if eligible_argument?(argument)
-        command.public_send(method, argument)
-      else
-        command.public_send(method)
-      end
+      command.public_send(method, argument) if eligible_argument?(argument)
```

Comment:
@georgeclaghorn I collapsed this down to one line and all of the tests pass, though I'm not entirely sure of the ramifications. I know you worked on this previously around supporting the `layers` option according to the git log, so if you happen to know if this breaks any current behavior, I'd be more than happy to write a new test for that and update this method accordingly.

Reply:
We take the else branch when the argument is `true` or otherwise present.

As far as I can tell, this change results in a different `mogrify` command. I’d preserve the existing structure.